test/norm_flows.jl

```diff
@@ -0,0 +1,24 @@
+using Test
+using Bijectors, ForwardDiff, LinearAlgebra
+
+@testset "planar flows" begin
+    for i in 1:10
+        flow = PlanarLayer(10)
+        z = randn(10,100)
+        forward_diff = log(abs(det(ForwardDiff.jacobian(t -> transform(flow, t), z))))
+        our_method = sum(forward(flow, z).logabsdetjacob)
+        @test our_method ≈ forward_diff
+        # @test inv(flow, transform(flow, z)) ≈ z
+    end
+end
+
+@testset "radial flows" begin
+    for i in 1:10
+        flow = RadialLayer(10)
+        z = randn(10,100)
+        forward_diff = log(abs(det(ForwardDiff.jacobian(t -> transform(flow, t), z))))
+        our_method = sum(forward(flow, z).logabsdetjacob)
+        @test our_method ≈ forward_diff
+        # @test inv(flow, transform(flow, z)) ≈ z
```

Comment:
@sharanry Why do we comment this out?

Reply:
For radial flows, it is working so i have kept it as a `@test`. For planar flows, I have now kept it as a `@test_broken` as the inverse is not accurate enough to pass with the `≈` operator.

Reply:
I will push a PR to your branch soon to fix this.